### PR TITLE
📊 Global Wildfire Information System - Monthly Burnt Areas dataset

### PR DESCRIPTION
**Pull request by veronikasamborska1994**

### CODE REVIEW

File: snapshots/climate/2024-02-19/monthly_fire_emissions.zip.dvc [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    # Files
+    url_main: https://gwis.jrc.ec.europa.eu/apps/country.profile/downloads
+    url_download: 
+      https://effis-gwis-cms.s3.eu-west-1.amazonaws.com/apps/country.profile/emission_gfed_full_2002_2022.zip
+    date_accessed: 2024-02-19
+
+    # License
+    license:
+      name: European Union, 1995-2023
+      url: https://gwis.jrc.ec.europa.eu/about-gwis/data-license
```
Comment (paarriagadap) [inline snapshots/climate/2024-02-19/monthly_fire_emissions.zip.dvc:26]: The link also mentions CC BY 4.0. Maybe it's better to use that one?

File: snapshots/climate/2024-02-19/monthly_burned_area.zip.dvc [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+    # Files
+    url_main: https://gwis.jrc.ec.europa.eu/apps/country.profile/downloads
+    url_download: 
+      https://effis-gwis-cms.s3.eu-west-1.amazonaws.com/apps/country.profile/MCD64A1_burned_area_full_dataset_2002-2022.zip
+    date_accessed: 2024-02-19
+
+    # License
+    license:
+      name: European Union, 1995-2023
```
Comment (paarriagadap) [inline snapshots/climate/2024-02-19/monthly_burned_area.zip.dvc:25]: Some comment about the license here

File: etl/steps/data/meadow/climate/2024-02-19/monthly_fire_emissions.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        "SO2",
+        "BC",
+        "NOx",
+    ]
+    tb = tb[columns_to_keep]
+    #
+    # Process data.
+    #
+    # Ensure all columns are snake-case, set an appropriate index, and sort conveniently.
+    tb = tb.underscore().set_index(["country", "year", "month", "region"], verify_integrity=True).sort_index()
```
Comment (paarriagadap) [inline etl/steps/data/meadow/climate/2024-02-19/monthly_fire_emissions.py:51]: Nitpick: underscore is also included in the create_dataset function

File: etl/steps/data/garden/climate/2024-02-19/monthly_fire_emissions.meta.yml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    presentation:
+      topic_tags:
+        - Climate Change
+    display:
+      numDecimalPlaces: 0
+      zeroDay: '2000-01-01'
+      yearIsDay: true
+    short_unit: tonnes
+    unit: t
+
```
Comment (paarriagadap) [inline etl/steps/data/garden/climate/2024-02-19/monthly_fire_emissions.meta.yml:13]: `processing_level` is missing

File: etl/steps/data/garden/climate/2024-02-19/monthly_fire_emissions.meta.yml [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/tables/
+tables:
+  monthly_fire_emissions:
+    # Learn more about the available fields:
+    # http://docs.owid.io/projects/etl/architecture/metadata/reference/indicator/
+    variables:
+      co2:
+        title: Monthly CO₂ emissions from fires
+        description_key:
+        - Monthly CO₂ emissions from fires measure the carbon dioxide released by fires each month.
```
Comment (paarriagadap) [inline etl/steps/data/garden/climate/2024-02-19/monthly_fire_emissions.meta.yml:32]: Maybe the first point in both variables can be written as a `description_short`

File: etl/steps/data/garden/climate/2024-02-19/monthly_burned_area.meta.yml [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+    presentation:
+      topic_tags:
+        - Climate Change
+    display:
+      numDecimalPlaces: 0
+      zeroDay: '2000-01-01'
+      yearIsDay: true
+    short_unit: ha
+    unit: hectares
+
```
Comment (paarriagadap) [inline etl/steps/data/garden/climate/2024-02-19/monthly_burned_area.meta.yml:13]: processing_level is missing

File: etl/steps/data/garden/climate/2024-02-19/monthly_burned_area.meta.yml [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/tables/
+tables:
+  monthly_burned_area:
+    # Learn more about the available fields:
+    # http://docs.owid.io/projects/etl/architecture/metadata/reference/indicator/
+    variables:
+      forest:
+        title: Monthly burned area across forests
```
Comment (paarriagadap) [inline etl/steps/data/garden/climate/2024-02-19/monthly_burned_area.meta.yml:29]: I would include at least description_short to explain what forest, savannas, shrublands, grasslands, etc are

File: etl/steps/data/grapher/climate/2024-02-19/yearly_burned_area.meta.yml [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+  common:
+    presentation:
+      topic_tags:
+        - Climate Change
+    display:
+      numDecimalPlaces: 0
+      yearIsDay: false
+    short_unit: ha
+    unit: hectares
+
```
Comment (paarriagadap) [inline etl/steps/data/grapher/climate/2024-02-19/yearly_burned_area.meta.yml:12]: processing_level is missing

File: etl/steps/data/grapher/climate/2024-02-19/yearly_fire_emissions.meta.yml [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+definitions:
+  common:
+    presentation:
+      topic_tags:
+        - Climate Change
+    display:
+      numDecimalPlaces: 0
+      yearIsDay: false
+    short_unit: tonnes
+    unit: t
```
Comment (paarriagadap) [inline etl/steps/data/grapher/climate/2024-02-19/yearly_fire_emissions.meta.yml:11]: processing_level is missing

File: etl/steps/data/grapher/climate/2024-02-19/yearly_burned_area.py [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+    grouped_tb = grouped_tb.set_index(["country", "year"], verify_integrity=True)
+
+    #
+    # Save outputs.
+    #
+    # Create a new grapher dataset with the same metadata as the garden dataset.
+    ds_grapher = create_dataset(
+        dest_dir, tables=[grouped_tb], check_variables_metadata=True, default_metadata=ds_garden.metadata
+    )
+    ds_grapher.metadata.title = "Global Yearly Burned Area"
```
Comment (paarriagadap) [inline etl/steps/data/grapher/climate/2024-02-19/yearly_burned_area.py:47]: You can also add this in the yaml: dataset.title

File: etl/steps/data/grapher/climate/2024-02-19/yearly_fire_emissions.py [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+    grouped_tb = grouped_tb.set_index(["country", "year"], verify_integrity=True)
+
+    #
+    # Save outputs.
+    #
+    # Create a new grapher dataset with the same metadata as the garden dataset.
+    ds_grapher = create_dataset(
+        dest_dir, tables=[grouped_tb], check_variables_metadata=True, default_metadata=ds_garden.metadata
+    )
+    ds_grapher.metadata.title = "Global Yearly Emissions from Fires"
```
Comment (paarriagadap) [inline etl/steps/data/grapher/climate/2024-02-19/yearly_fire_emissions.py:47]: You can also add this in the yaml: dataset.title

**Review by paarriagadap — APPROVED** [review]
Good! Besides the comments below you also need to run make check to delete unused imports and delete an empty Jupyter notebook in meadow/un/2023-01-24

File: snapshots/climate/2024-02-19/monthly_fire_emissions.zip.dvc (hunk 1 above)
Comment (veronikasamborska1994) [reply snapshots/climate/2024-02-19/monthly_fire_emissions.zip.dvc:26]: ah yeah thanks! nicely spotted :)

File: etl/steps/data/garden/climate/2024-02-19/monthly_fire_emissions.meta.yml (hunk 4 above)
Comment (veronikasamborska1994) [reply etl/steps/data/garden/climate/2024-02-19/monthly_fire_emissions.meta.yml:13]: thanks!

File: etl/steps/data/grapher/climate/2024-02-19/yearly_burned_area.py (hunk 10 above)
Comment (veronikasamborska1994) [reply etl/steps/data/grapher/climate/2024-02-19/yearly_burned_area.py:47]: good point! thanks!

File: etl/steps/data/garden/climate/2024-02-19/monthly_burned_area.meta.yml (hunk 7 above)
Comment (veronikasamborska1994) [reply etl/steps/data/garden/climate/2024-02-19/monthly_burned_area.meta.yml:29]: thanks! :)

**Pull request merged**